tellus-guestvm-dir/create_guest_image.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
```

Comment:
Reuse existing script instead of duping it.

Reply:
fixed